build/build.c

```diff
@@ -228,12 +229,13 @@ static rpmRC doCheckBuildRequires(rpmts ts, rpmSpec spec, int test)
     rpmps ps = rpmSpecCheckDeps(ts, spec);
 
     if (ps) {
-	rpmlog(RPMLOG_ERR, _("Failed build dependencies:\n"));
-	rpmpsPrint(NULL, ps);
-    }
-    if (ps != NULL)
+	if (rpmIsVerbose()) {
+	    rpmlog(RPMLOG_ERR, _("Failed build dependencies:\n"));
```

Comment:
This newly added condition is wrong, errors and warnings must be emitted in --quiet mode too.

Reply:
Fixed!